images/builder/run.sh

```diff
@@ -22,9 +22,11 @@ echo "Activating service account..."
 gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
 
 echo "Running..."
-if [[ ! -z "${ARTIFACTS}" ]]; then
-  echo "\$ARTIFACTS is set, sending logs to ${ARTIFACTS}"
```

Comment:
This is still going to do nothing `$ARTIFACTS` and `$LOG_TO_STDOUT` are both set. You should combine the two conditions into a single statement.

Reply:
@Katharine -- moved them into a single statement.